src/templates/dandelion/components/TokenSelector/TokenSelectorInstance.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import styled from 'styled-components'
+import { addressesEqual, GU } from '@aragon/ui'
+import { shortenAddress } from '../../../../web3-utils'
+import { ETHER_TOKEN_FAKE_ADDRESS } from '../../config/helpers/tokens'
+
+/* eslint-disable react/prop-types */
+const TokenSelectorInstance = React.memo(function TokenSelectorInstance({
+  address,
+  name,
+  symbol,
+  showIcon = true,
+}) {
+  return (
+    <div
+      css={`
+        display: flex;
+        align-items: center;
+      `}
+    >
+      {showIcon ? (
+        <Icon src={`https://chasing-coins.com/coin/logo/${symbol}`} />
```

Comment:
If we have the token address, let's use `tokenIconUrl()` from [aragonUI](https://github.com/aragon/aragon-ui/blob/master/src/utils/web3.js#L187)